integration_test/tests/wallet.rs

```diff
@@ -36,6 +36,14 @@ fn wallet__abandon_transaction() {
     node.client.abandon_transaction(txid).expect("abandontransaction");
 }
 
+#[test]
+fn wallet__abort_rescan() {
+    let node = Node::with_wallet(Wallet::Default, &[]);
+
+    let json: AbortRescan = node.client.abort_rescan().expect("abortrescan");
+    assert!(json.0 == false);
```

Comment:
```suggestion
    assert!(!json.0); // No rescan running, abort should return false
```

It was not clear to me why this should be false and had to look it up. Therefore the comment.

Reply:
Alright. Noted. Thank you!